xbmc/input/actions/ActionTranslator.cpp

```diff
@@ -10,262 +10,263 @@
-    {"playpvr", ACTION_PVR_PLAY},
-    {"playpvrtv", ACTION_PVR_PLAY_TV},
-    {"playpvrradio", ACTION_PVR_PLAY_RADIO},
-    {"record", ACTION_RECORD},
-    {"togglecommskip", ACTION_TOGGLE_COMMSKIP},
-    {"showtimerrule", ACTION_PVR_SHOW_TIMER_RULE},
-    {"channelnumberseparator", ACTION_CHANNEL_NUMBER_SEP},
+     // PVR actions
+     {"channelup", ACTION_CHANNEL_UP},
+     {"channeldown", ACTION_CHANNEL_DOWN},
+     {"previouschannelgroup", ACTION_PREVIOUS_CHANNELGROUP},
+     {"nextchannelgroup", ACTION_NEXT_CHANNELGROUP},
+     {"playpvr", ACTION_PVR_PLAY},
+     {"playpvrtv", ACTION_PVR_PLAY_TV},
+     {"playpvrradio", ACTION_PVR_PLAY_RADIO},
+     {"record", ACTION_RECORD},
+     {"togglecommskip", ACTION_TOGGLE_COMMSKIP},
+     {"showtimerrule", ACTION_PVR_SHOW_TIMER_RULE},
+     {"channelnumberseparator", ACTION_CHANNEL_NUMBER_SEP},
 
-    // Mouse actions
-    {"leftclick", ACTION_MOUSE_LEFT_CLICK},
-    {"rightclick", ACTION_MOUSE_RIGHT_CLICK},
-    {"middleclick", ACTION_MOUSE_MIDDLE_CLICK},
-    {"doubleclick", ACTION_MOUSE_DOUBLE_CLICK},
-    {"longclick", ACTION_MOUSE_LONG_CLICK},
-    {"wheelup", ACTION_MOUSE_WHEEL_UP},
-    {"wheeldown", ACTION_MOUSE_WHEEL_DOWN},
-    {"mousedrag", ACTION_MOUSE_DRAG},
-    {"mousedragend", ACTION_MOUSE_DRAG_END},
-    {"mousemove", ACTION_MOUSE_MOVE},
+     // Mouse actions
+     {"leftclick", ACTION_MOUSE_LEFT_CLICK},
+     {"rightclick", ACTION_MOUSE_RIGHT_CLICK},
+     {"middleclick", ACTION_MOUSE_MIDDLE_CLICK},
+     {"doubleclick", ACTION_MOUSE_DOUBLE_CLICK},
+     {"longclick", ACTION_MOUSE_LONG_CLICK},
+     {"wheelup", ACTION_MOUSE_WHEEL_UP},
+     {"wheeldown", ACTION_MOUSE_WHEEL_DOWN},
+     {"mousedrag", ACTION_MOUSE_DRAG},
+     {"mousedragend", ACTION_MOUSE_DRAG_END},
+     {"mousemove", ACTION_MOUSE_MOVE},
 
-    // Touch
-    {"tap", ACTION_TOUCH_TAP},
-    {"longpress", ACTION_TOUCH_LONGPRESS},
-    {"pangesture", ACTION_GESTURE_PAN},
-    {"zoomgesture", ACTION_GESTURE_ZOOM},
-    {"rotategesture", ACTION_GESTURE_ROTATE},
-    {"swipeleft", ACTION_GESTURE_SWIPE_LEFT},
-    {"swiperight", ACTION_GESTURE_SWIPE_RIGHT},
-    {"swipeup", ACTION_GESTURE_SWIPE_UP},
-    {"swipedown", ACTION_GESTURE_SWIPE_DOWN},
+     // Touch
+     {"tap", ACTION_TOUCH_TAP},
+     {"longpress", ACTION_TOUCH_LONGPRESS},
+     {"pangesture", ACTION_GESTURE_PAN},
+     {"zoomgesture", ACTION_GESTURE_ZOOM},
+     {"rotategesture", ACTION_GESTURE_ROTATE},
+     {"swipeleft", ACTION_GESTURE_SWIPE_LEFT},
+     {"swiperight", ACTION_GESTURE_SWIPE_RIGHT},
+     {"swipeup", ACTION_GESTURE_SWIPE_UP},
+     {"swipedown", ACTION_GESTURE_SWIPE_DOWN},
 
-    // Voice
-    {"voicerecognizer", ACTION_VOICE_RECOGNIZE},
+     // Voice
+     {"voicerecognizer", ACTION_VOICE_RECOGNIZE},
 
-    // Do nothing / error action
-    {"error", ACTION_ERROR},
-    {"noop", ACTION_NOOP}};
+     // Do nothing / error action
+     {"error", ACTION_ERROR},
+     {"noop", ACTION_NOOP}});
```

Comment:
Does clang-format insert these spaces? Can we avoid clobbering history? Ideally without `// clang-format off` but maybe it's necessary.